website/docs/r/container_node_pool.html.markdown

```diff
@@ -149,7 +149,7 @@ cluster.
     [google_container_cluster](container_cluster.html) for schema.
 
 * `node_count` - (Optional) The number of nodes per instance group. This field can be used to
-    update the number of nodes per instance group but should not be used alongside `autoscaling`.
+    update the number of nodes per instance group.
```

Comment:
As per the documentation below, `node_count` is used alongside `autoscaling` and represents the starting point of the pool's size.

https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler#minimum_and_maximum_node_pool_size

Reply:
Yes, but in a declarative config system like Terraform, this will produce a diff if the autoscaler changes the node count.  We'll take the other change, though!  This repository is generated, though, so if you don't mind adding it to GoogleCloudPlatform/magic-modules, under third_party/terraform/website, that would be great.